### PR TITLE
refactor: periphery contracts refactor/cleanup

### PR DESCRIPTION
**Pull request by mds1**
- Configure and run `forge fmt`
- Check formatting in CI
- Use named imports
- Remove extra `balanceOf` check in `UniswapWithdrawHook.tokensWithdrawn()`

### CODE REVIEW

**Review by garyghayrat — APPROVED** [review]

**Review by apbendi — COMMENTED** [review]
Looks good to me except for the uint change!

**Comment by mds1** [comment]
@apbendi re `forge fmt` changes, are you on the latest forge version? Given the CI passes and it uses the most recent version I suspect yours may be outdated (unless forge fmt changed since CI was ran)

File: contracts-periphery/src/UmbraBatchSend.sol [hunk 1]
```diff
@@ -1,74 +1,82 @@
...
     bytes32 pkx;
     bytes32 ciphertext;
   }
 
-  mapping(address => uint256) internal totalTransferAmountPerToken;
+  mapping(address => uint) internal totalTransferAmountPerToken;
+
   struct TransferSummary {
-    uint256 amount;
+    uint amount;
```
[the remark this answers is not in the document]
Comment (apbendi) [reply contracts-periphery/src/UmbraBatchSend.sol:33]: Ooof I know it's a bikeshed but I *really* **really** hate this 🙃. Also goes against the way we've written all our solidity until now. Unless there's very strong consensus in favor of `uint` from the rest of the team I'm inclined to ask we change this.

File: contracts-periphery/test/UmbraBatchSend.t.sol [hunk 2]
```diff
@@ -1,14 +1,16 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.13;
 
-import "test/utils/DeployUmbraTest.sol";
-import "src/UmbraBatchSend.sol";
+import {IERC20} from "openzeppelin-contracts/token/ERC20/IERC20.sol";
+import {DeployUmbraTest} from "test/utils/DeployUmbraTest.sol";
+import {IUmbra}  from"src/interface/IUmbra.sol";
```
Comment (apbendi) [inline contracts-periphery/test/UmbraBatchSend.t.sol:6]: I did a `foundryup` but running `forge fmt` locally still results in changes. The changes seem to be related to the formatting of imports, for example fixing the spacing here, which does seem wrong. Weirdly, when I re-run the CI job it passes. Not sure what's up 🤔

Comment (mds1) [reply contracts-periphery/test/UmbraBatchSend.t.sol]: I just updated forge, committed changes from `forge fmt`, and CI passes too. Lmk if your `forge fmt` matches now

File: contracts-periphery/test/gas/UmbraBatchSend.gas.t.sol [hunk 3]
```diff
@@ -37,12 +40,9 @@ abstract contract UmbraBatchSendGasTest is DeployUmbraTest {
     assertTrue(ethBalance > 0 && tokenBalance > 0);
   }
 
-  function executeParams(
-    Send _type,
-    uint256 numOfAddrs,
-    uint256 etherAmount,
-    uint256 tokenAmount
-  ) public {
+  function executeParams(Send _type, uint256 numOfAddrs, uint256 etherAmount, uint256 tokenAmount)
```
[the remark this answers is not in the document]
Comment (mds1) [reply contracts-periphery/test/gas/UmbraBatchSend.gas.t.sol:43]: There isn't, thankfully, cause forcing that all the time would be annoying IMO haha 😛

AFAIK all formatters (well at least rustfmt and prettier which I'm most familiar with) will always put the full function signature (or any full statement) on one line if it fits with the line-length (and sometimes same line if its 1 char over too)

**Review by apbendi — APPROVED** [review]

**Pull request merged**